docs/en/enterprise-edition/policy-reference/aws-policies/aws-iam-policies/bc-aws-355.adoc

```diff
@@ -1,5 +1,5 @@
 
-== IAM policy document allows "*" as a resource for any action that can be restricted
+== IAM policy document allows all resources with restricted actions
```

Comment:
IAM policy allows all resources with restricted actions

Reply:
This is a Prisma defined name, unfortunately.